### ISSUE TITLE
Number keyboard overlaying the question on tablets

### ISSUE THREAD

**Issue by janagombitova** [akvo/akvo-flow-mobile#472]
On tablets (Samsung Tab 3 tested) if a number questions was at the end of a question group the screen would not adjust. Therefore the number pad would cover the question and answer so users couldn't see what they were writing. 

This does not happen on the phones, but was an issue for tablets.

**Comment by janagombitova** [comment]
Created via Reamaze:

Link: https://flowhelp.reamaze.com/admin/conversations/akvoflow-support-request-number-29356-from-nikki-at-akvo-dot-org

**Comment by janagombitova** [comment]
<img width="318" alt="screen shot 2016-11-08 at 16 03 10" src="https://cloud.githubusercontent.com/assets/12456965/20104078/e2fb18a4-a5cc-11e6-9762-33a51ea5f6f5.png">
*view on how the last question on the screen looks like*

**Comment by valllllll2000** [comment]
![number-display-1](https://cloud.githubusercontent.com/assets/923280/20482746/8c554660-afee-11e6-9d38-48ccc11cdc79.png)

The issue does not happen on my sony tablet with android 6.0.1

**Comment by valllllll2000** [comment]
I tried to reproduce it on all my devices and emulators and i just could not reproduce it. We should check if this happened on other devices but for now I cannot fix it if i cannot reproduce. Moreover attempting to fix it on just one device may have undesirable effects on other devices.

**Comment by janagombitova** [comment]
@valllllll2000 I tried to replicate the issue on Tab 3, running Android 4.4.2 and here is how I saw the issue. The question did not move nicely above the keyboard as in your example. Can it be linked to the Android version? 

![screenshot_2016-11-25-14-33-50](https://cloud.githubusercontent.com/assets/12456965/20626826/728e6b0e-b31c-11e6-89d1-12d9d613a1b5.png)
![screenshot_2016-11-25-14-33-31](https://cloud.githubusercontent.com/assets/12456965/20626827/728eb514-b31c-11e6-802b-c0464551ffad.png)
